materialize-mysql/sqlgen.go

```diff
@@ -163,7 +163,7 @@ type templates struct {
 	updateLoad        *template.Template
 	updateReplace     *template.Template
 	updateTruncate    *template.Template
-	storeLoad         *template.Template
+	insertLoad        *template.Template
```

Comment:
It's a bit of extra term, but the variable name `storeLoad` was absolutely breaking my brain.